request.c

```diff
@@ -404,6 +416,13 @@ void getTopClients(char *client_message, int *sock)
 		count = num;
 	}
 
+	// Apply Audit Log filtering?
+	bool includezeroclients = false;
+	if(command(client_message, " withzero"))
```

Comment:
This option can be combined with existing options, i.e. both `>top-clients withzero` and `>top-clients withzero (123)` are valid